src/main/java/com/blogify/blogapi/model/User.java

```diff
@@ -0,0 +1,24 @@
+package com.blogify.blogapi.model;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.time.LocalDate;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.ToString;
+
+@Data
+@Builder
+@ToString
+@AllArgsConstructor
+public class User implements Serializable {
```

Comment:
add @EqualsAndHashCode
add @Entity
i forget if @data have @ToString in. if not, add @ToString
better to have @Table(name = "\".....\"")